core/engine/src/vm/tests.rs

```diff
@@ -498,3 +492,20 @@ fn long_object_chain_gc_trace_stack_overflow() {
         TestAction::inspect_context(|_| boa_gc::force_collect()),
     ]);
 }
+
+// See: https://github.com/boa-dev/boa/issues/4515
+#[test]
```

Comment:
thought: do we have a notation for marking these tests as TODO?

I think linking the issue is good, but it might be nice to add a `FIX:` or `TODO (fix):` prefix to tests like these so that they can be easily grepped. But maybe `#[ignore]` is enough.